prow/cmd/deck/static/prow/prow.ts

```diff
@@ -747,7 +740,7 @@ function createViewJobCell(prowjob: string): HTMLTableDataCellElement {
     const c = document.createElement("td");
     const i = icon.create("pageview", "Show job YAML", () => gtag("event", "view_job_yaml", {event_category: "engagement", transport_type: "beacon"}));
     i.href = `/prowjob?prowjob=${prowjob}`;
-    c.classList.add("icon-cell");
+    c.style.width = "75px"
```

Comment:
is this intended? hardcoded 75px doesn't seem mobile friendly

Reply:
Currently icon-cell is a hardcoded 32px, with no handling of mobile screen sizes. I followed this pattern.

Reply:
1) Does it need a `width` style at all? (If you don't specify any, it should auto-fit, right??)

2) If anything, I'd say add column-sizing classes to the table _headers_ would be the best option.

3) I'd also strongly, strongly, strongly recommend using a CSS class (vs direct element styles). If any is needed, of course.

Reply:
1. It will auto-fit but the text in header will stack 
       Job                        vs                 Job YAML
       YAML
     But I realized that removing the class in itself we get desired change
 2. Fixed! This change definitely helps out with code simplicity.
 3. Fixed!